examples/serenity/voice/src/main.rs

```diff
@@ -63,14 +63,15 @@ async fn main() {
         .await
         .expect("Err creating client");
 
-    let _ = client.start().await.map_err(|why| println!("Client ended: {:?}", why));
+    if let Err(why) = client.start().await {
+        println!("Err starting client: {:?}", why))
+    };
 }
 
 #[command]
 #[only_in(guilds)]
 async fn deafen(ctx: &Context, msg: &Message) -> CommandResult {
-    let guild = msg.guild(&ctx.cache).await.unwrap();
-    let guild_id = guild.id;
+    let guild_id = msg.guild_id;
```

Comment:
`Message::guild_id` can be `None` if the message came from a DM, but in this instance it is guarded against with the `#[only_in(guilds)]` precondition check, so this should be `unwrap`ed:
```suggestion
    let guild_id = msg.guild_id.unwrap();
```

Reply:
isnt there any other way guild_id is none? maybe some bug i dont think it requires panicking so i'd just return if its None